examples/thermostat/thermostat-common/include/thermostat-delegate-impl.h

```diff
@@ -50,6 +50,8 @@ static constexpr uint8_t kMaxNumberOfPresetsSupported = kMaxNumberOfPresetTypes
 
 static constexpr uint8_t kMaxNumberOfSchedulesSupported = kMaxNumberOfScheduleTypes * kMaxNumberOfSchedulesOfEachType - 1;
 
+static constexpr uint8_t kMaxNumberOfScheduleTransitionsPerDay = 4;
```

Comment:
Attribut name is `NumberOfScheduleTransitionPerDay`:
```suggestion
static constexpr uint8_t kMaxNumberOfScheduleTransitionPerDay = 4;
```